src/Microsoft.AspNet.OData.Shared/EdmEntityObject.cs

```diff
@@ -6,15 +6,19 @@
 //------------------------------------------------------------------------------
 
 using System;
+using Microsoft.AspNet.OData.Builder;
+using Microsoft.AspNet.OData.Common;
 using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+using Org.OData.Core.V1;
 
 namespace Microsoft.AspNet.OData
 {
     /// <summary>
     /// Represents an <see cref="IEdmEntityObject"/> with no backing CLR <see cref="Type"/>.
     /// </summary>
     [NonValidatingParameterBinding]
-    public class EdmEntityObject : EdmStructuredObject, IEdmEntityObject
+    public class EdmEntityObject : EdmStructuredObject, IEdmEntityObject, IEdmChangedObject
```

Comment:
For my understanding, why was this type inheriting from `IEdmChangedObject` before and why does it not need to do that now? Was it a bug?

Reply:
The type was not inheriting from `IEdmChangedObject` before but it is now. Is that what you meant? It has been added and not removed?